src/Symfony/Bridge/Twig/Extension/TranslationExtension.php

```diff
@@ -125,7 +125,7 @@ public function trans($message, array $arguments = [], string $domain = null, st
         return $this->getTranslator()->trans($message, $arguments, $domain, $locale);
     }
 
-    public function createTranslatable(string $message, array $parameters = [], string $domain = 'messages'): Translatable
+    public function createTranslatable(string $message, array $parameters = [], string $domain = null): Translatable
```

Comment:
in `trans()`, shouldn't we support using the `$domain` argument explicitly when the `Translatable` object does not have a domain ? Currently, we always override the domain argument on line 113.

Reply:
L115 updated to `$domain = $domain ?? $message->getDomain();`